node/node.go

```diff
@@ -667,9 +664,16 @@ func (node *Node) Start() error {
 						context.TODO(), openBox,
 					)
 					if err != nil {
-						// TODO (lc): block peers sending error messages
-						errChan <- withError{err, msg.GetFrom()}
-						return libp2p_pubsub.ValidationReject
+						switch err {
+						case errIgnoreBeaconMsg:
+							// ignore the further processing of the ignored messages as it is not intended for this node
+							// but propogate the messages to other nodes
+							return libp2p_pubsub.ValidationAccept
```

Comment:
Is it better to be `ValidationIgnore` so that the node will forward ignored message?